src/store/data-slice.js

```diff
@@ -0,0 +1,71 @@
+import { createSlice} from '@reduxjs/toolkit';
```

Comment:
Code format doesn't look pretty and neat(ex. indentations don't match) you can use your idea reformat

Reply:
installed prettier.issue fixed.